Utilities/KWStyle/KWStyle.cmake

```diff
@@ -4,10 +4,10 @@ if(ITK_USE_KWSTYLE)
     ${ITK_SOURCE_DIR}/Utilities/KWStyle/ITK.kws.xml
     )
   set(kwstyle_itk_examples_files_list_file
-    ${ITK_BINARY_DIR}/Utilities/KWStyle/ITKExamplesFiles.txt
+    ${ITK_BINARY_DIR}/Utilities/KWStyle/ITKSphinxExamplesFiles.txt
```

Comment:
Same comment for the contents of that will populate this text file.

Reply:
:+1: good catch.